src/main/java/core/model/HOModel.java

```diff
@@ -100,29 +103,25 @@ public final void setAllOldSpieler(Vector<Player> playerVector) {
     /**
      * Gibt alle alten Player (nicht mehr im Team befindliche) zurück
      */
-    public final Vector<Player> getAllOldSpieler() {
+    public final List<Player> getAllOldSpieler() {
+    	if ( m_vOldPlayer == null){
+    		m_vOldPlayer = DBManager.instance().getAllSpieler();
+		}
         return m_vOldPlayer;
     }
 
     //---------Player--------------------------------------
 
     /**
-     * Gibt alle Player zurück
+     * Returns all current players
      */
-    public final Vector<Player> getAllSpieler() {
+    public final List<Player> getCurrentPlayer() {
```

Comment:
This should be made plural, `getCurrentPlayers`, it's confusing otherwise.

Reply:
getCurrentPlayers would be my favorite
there is another getter getAllOldSpieler (sorry i missed this one) which fetches the complete list of historical players of your club. getAllPlayers could lead to confusion i think.